src/realm/obj.hpp

```diff
@@ -300,10 +297,12 @@ class Obj : public CollectionParent {
     LnkSetPtr get_linkset_ptr(ColKey col_key) const;
     SetBasePtr get_setbase_ptr(ColKey col_key) const;
     Dictionary get_dictionary(ColKey col_key) const;
-    DictionaryPtr get_dictionary_ptr(ColKey col_key) const;
-    std::shared_ptr<Dictionary> get_dictionary_ptr(const std::vector<CollectionParent::Index>& path) const;
     Dictionary get_dictionary(StringData col_name) const;
 
+    DictionaryPtr set_dictionary_ptr(ColKey col_key);
+    DictionaryPtr get_dictionary_ptr(ColKey col_key) const;
+    // std::shared_ptr<Dictionary> get_dictionary_ptr(const std::vector<CollectionParent::Index>& path) const;
```

Comment:
Commented code. It seems no longer needed.

Reply:
Removed